src/reputation/distributions.py

```diff
@@ -16,6 +23,90 @@ def amount(self):
     def gives_rep(self):
         return self._give_rep
 
+RSC_YEARLY_GIVEAWAY = 50000000
+MINUTES_IN_YEAR = 525960
+HOURS_IN_YEAR = MINUTES_IN_YEAR / 60
+DAYS_IN_YEAR = 365
+MONTHS_IN_YEAR = 12
+GROWTH = .2
+
+def calculate_upvote_rsc():
```

Comment:
Just a suggestion, had to dig into one this method does. I think it calculates the RSC to distribute per one vote?
If that's the case, perhaps we can rename to something like `calculate_rsc_per_upvote` or `calculate_rsc_to_distribute_per_upvote` or `calc_rsc_per_upvote` or something like that